src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Circle extends Figure {
+    private int radius = 0;
```

Comment:
init this value inside of the constructor, but don't generate a random value here, it should be passed from the outside to constructor